client/index.js

```diff
@@ -0,0 +1,5 @@
+import Banner from './components/banner.jsx';
+import TrailDescription from './components/traildescription.jsx';
+
+window.Banner = Banner;
+window.TrailDescription = TrailDescription;
```

Comment:
recommend namespacing!

Reply:
@cjm771 - agreed. I'll implement that `let NT = window.NT || {};` in a to do ticket. Thanks for that, great idea.